src/pages/App.scss

```diff
@@ -5,9 +5,10 @@
     height: 100%;
     position: relative;
 
-    @media only screen and (min-device-width : 768px) {
+    @media only screen and (min-width : 768px) {
       margin: auto;
-      width: 560px;
+      max-width: 560px;
```

Comment:
this and the line below are mostly responsible for the better responsiveness, the min-device-width to min-width is best practices